src/components/Post/GovernanceSideBar/Bounty/Curator/SubmissionAction.tsx

```
@@ -0,0 +1,149 @@
+		await handleDelete(submission);
+		setLoading(false);
+	};
+
+	return (
+		<>
+			<div className='flex w-full'>
+				{submission?.bountyData?.curator === encodedAddress && submission?.status === EChildbountySubmissionStatus.PENDING ? (
+					<>
+						{' '}
+						<span
+							onClick={() => showRejectModal && showRejectModal(submission)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)